src/parser.c

```diff
@@ -225,25 +234,33 @@ int ecs_parse_component_expr(
                 ecs_system_expr_oper_kind_t prev_oper_kind = oper_kind;
                 bptr = parse_complex_elem(bptr, &elem_kind, &oper_kind, &source);
                 if (!bptr) {
-                    ecs_abort(ECS_INVALID_EXPRESSION, sig);
+                    char err[50] = "/0";
```

Comment:
If this happens, an error was detected in the `parse_complex_elem` function. The error should be printed there since we have more information there about what went wrong.

Reply:
I tried to do it, but It seems to me that I have to pollute the system.c API in order to give the information to communicate que error. Maybe there is a way, but I'd prefer to leave it for later, once I have a better mental image of the project. For now I just tried to touch only the parser. If you have a idea about how to handle errors at system level, I'm open for suggestions.